target_postgres/sql_base.py

```diff
@@ -1023,21 +1027,72 @@ def write_batch_helper(self, connection, root_table_name, schema, key_properties
         :return: {'records_persisted': int,
                   'rows_persisted': int}
         """
+        batch__timing_start = datetime.datetime.now()
         records_persisted = len(records)
+
+        self.LOGGER.info('Writing batch: {}'.format(
+            json.dumps({'root_table_name': root_table_name,
+                        'key_properties': key_properties,
+                        'records_count': records_persisted})
+        ))
+
         rows_persisted = 0
         for table_batch in self._get_table_batches(schema, key_properties, records):
             table_batch['streamed_schema']['path'] = (root_table_name,) + table_batch['streamed_schema']['path']
+
+            table_batch__schema__timing_start = datetime.datetime.now()
+
+            self.LOGGER.info('Writing table batch schema: {}'.format(
+                json.dumps({'path': table_batch['streamed_schema']['path']})
+            ))
+
             remote_schema = self.upsert_table_helper(connection,
                                                      table_batch['streamed_schema'],
                                                      metadata)
-            rows_persisted += self.write_table_batch(
+
+            self.LOGGER.info('Table batch schema written: {}'.format(
```

Comment:
Dido, more human readable, metrics with tags will give an opportunity for something parable within the Singer framework